docs/en/ingest-management/release-notes/release-notes-8.12.asciidoc

```diff
@@ -35,6 +35,31 @@ Review important information about {fleet-server} and {agent} for the 8.12.0 rel
 {agent}::
 * Update Go version to 1.20.12. {agent-pull}3885[#3885]
 
+[discrete]
+[[known-issues-8.12.0]]
+=== Known issues
+
+[[known-issue-4084]]
+.`elastic-agent enroll` command incorrectly reports failure
```

Comment:
```suggestion
.For new DEB and RPM installations the `elastic-agent enroll` command incorrectly reports failure
```

Let's put the DEB and RPM qualifier directly in the title.

Reply:
Fixed in [f517ede](https://github.com/elastic/ingest-docs/pull/773/commits/f517ede6c761d6c475256824149cd3717bacae2d)